xesar-connect/src/main/kotlin/com/open200/xesar/connect/messages/SetPhoneNumberOnSmartphoneResult.kt

```diff
@@ -0,0 +1,19 @@
+package com.open200.xesar.connect.messages
+
+import com.open200.xesar.connect.messages.event.MediumChanged
+import com.open200.xesar.connect.messages.event.PhoneNumberChanged
+import java.util.*
+import kotlinx.coroutines.Deferred
+import kotlinx.coroutines.awaitAll
+
+/**
+ * Pair of events returning from
+ * [com.open200.xesar.connect.extension.setPhoneNumberOnSmartphoneAsync]
```

Comment:
can you add the params to the java docs?

Reply:
done :)